src/state-machines/index.html

```diff
@@ -0,0 +1,188 @@
+﻿<!--
```

Comment:
I may have mentioned the wrong index.html. Instead of this one, I think you need to copy `blockly-state-machines/Blockly/index.html` from your other branch.